src/application-base/ApplicationBase.jsx

```diff
@@ -105,50 +80,26 @@ const ApplicationBase = ({
     return () => {
       window.removeEventListener('beforeunload', onBeforeUnload);
     };
-  }, [unloadPromptIsDisabled, registeredPromptsRef]);
-
-  const { localeOverride } = useTestOverrides(); // Allows us to test deployed applications in different locales.
-
-  const theme = useMemo(() => ({
-    // If the theme class name is undefined or an empty string, that indicates we have the root theme and should apply the root theme name.
-    name: themeName || rootThemeName,
-    className: themeName,
-  }), [themeName]);
+  }, [unloadPromptIsDisabled]);
 
   return (
-    <div data-terra-application-base className={cx('application-base', { fill: !fitToParentIsDisabled })}>
-      <ThemeProvider
-        themeName={themeName}
-      >
-        <ThemeContextProvider theme={theme}>
-          <Base
-            customMessages={customTranslatedMessages}
-            translationsLoadingPlaceholder={translationsLoadingPlaceholder}
-            locale={localeOverride || locale || browserLocale}
-          >
-            <ApplicationErrorBoundary>
```

Comment:
Why remove the error boundary? That feels like a pretty core thing that even embedded apps may want or even other use cases like apps transitioning from legacy architectures that mix legacy UI frameworks with Terra components mounted on the page instead of a single React root.

Reply:
That's a good point, those are definitely scenarios we want to support. But to be clear, the expectation is not that consumers of the framework will now need to provide their own error boundaries. Error boundaries will be provided by the framework and integrated within some of the new components being added to v2. I will call out where these are being added in future PRs to ensure we're on the same page.

Reply:
Ok I'll look for that. My main interest here is that apps transitioning from legacy architectures that are mounting React components onto DOM in existing pages can use `ApplicationBase` as a bootstrap for all the core things a single page app may have so that they can easily transition these separately mounted components into a single React root down the road without rework.